src/core/thread/address_resolver.cpp

```diff
@@ -415,7 +367,9 @@ void AddressResolver::HandleAddressError(Coap::Header &aHeader, Message &aMessag
 
     if (!aMessageInfo.GetSockAddr().IsMulticast())
     {
-        SendAddressErrorResponse(aHeader, aMessageInfo);
+        SuccessOrExit(mCoapServer.SendEmptyResponse(aHeader, aMessageInfo));
```

Comment:
I think we still want to process this Address Error message, even if we failed to send a response.